drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/LocalBinarySource.java

```diff
@@ -40,4 +41,9 @@ public ExternalBinary getReleaseForVersion(String version) throws Exception {
             return getLatestRelease();
         }
     }
+
+    @Override
+    public String getFileNameRegexToDownload(String version) {
+        return null;
```

Comment:
Well, in this case (test) it doesn't matter, but I agree that `null` may look add - the regex `".*"` could be more convenient